src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/PrisonerWingCalculator.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.utils
+
+import uk.gov.justice.digital.hmpps.visitscheduler.dto.PrisonerDetailDto
+
+/**
+ * this is only a temporary solution till we can get it from an API
+ */
+class PrisonerWingCalculator {
```

Comment:
It's not a Calculator it's a Parser,....
In fact I would move all this into a service.PrisionApiService
have the getPrisonerWing in this service.
Then just parse the wing in the service you could leave it as an internal class